specs/protocol/configurability.md

```diff
@@ -0,0 +1,71 @@
+| Fee margin                            | Markup on transactions compared to the raw L1 data cost.                                                                     |
+| Fee vault contracts                   | Contracts which dictate how user fees are distributed.                                                                       |
+| Gas limit                             | L2 block gas limit.                                                                                                          |
+| Gas Token                             | Native token used to pay for gas.                                                                                            |
+| Genesis state                         | Initial state at chain genesis, including code and storage of predeploys (all L2 smart contracts). See [Predeploy](../glossary.md#l2-genesis-block).                                            |
+| L1 smart contracts                    | The chain’s L1 smart contracts.                                                                                              |
+| L2 block time                         | Frequency with which blocks are produced as a result of derivation.                                                          |
+| Output frequency                      | Frequency with which output roots are submitted to L1.                                                                       |
+| Resource config                       | Config for the EIP-1559 based curve used for the deposit gas market.                                                         |
+| Sequencing window                     | Maximum allowed batch submission frequency, after which L1 fallback is triggered in derivation.                                                                                                                                                            |
+| Start block                           | Block at which op-node starts searching for logs from.                                                                       |
+
+## Admin Roles
+
+| Config Property                       | Description                                                                                                                  |
+|---------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+| L1 Proxy Admin                        | Account authorized to upgrade L1 contracts.                                                                                  |
+| L1 ProxyAdmin owner                   | Account authorized to update the L1 Proxy Admin.                                                                             |
+| L2 Proxy Admin                        | Account authorized to upgrade L2 contracts.                                                                                  |
```

Comment:
What do you mean by "contracts" here? Do you mean contracts owned by L2 Proxy admin? ie all predeploys